interface/src/app/utils.js

```diff
@@ -17,4 +17,24 @@ angular.module('adage.utils', [])
 
 // Number of digits in activity value shown on frontend:
 .constant('ActivityDigits', 5)
+
+// Service of global machine learning model, which is shared by multiple
+// modules.
```

Comment:
If I want to add a feature or fix a bug related to the `MLModelTracker`, the logical place to look for that code is in the `app/mlmodel` directory. I would never think to look in `app/utils.js`. This is part of what I was trying to advocate for in our discussion last Wednesday.

There is nothing wrong with declaring that `adage.utils` depends upon the module that contains `MLModelTracker` *and* having each page include the same dependency. The Angular dependency injection system has no trouble sorting that out.

Doing it the way I suggest will help us to avoid bugs like the `'accession' field does not allow filtering` one we found earlier. The structure of our repository should be intuitive enough that every developer knows without question which part of the source tree holds what code.